packages/boilerplate-generator/boilerplate_web_cordova_template.js

```diff
@@ -0,0 +1,70 @@
+// Template function for rendering the boilerplate html for cordova
+// Replicates the template defined in boilerplate_web.cordova.html
+// Arguments: root : { htmlAttributes, css : [{ url }], bundledJsCssUrlRewriteHook : Function, head, dynamicHead, body, dynamicBody, inlineScriptsAllowed, additionalStaticJs, meteorRuntimeConfig }
+
+export default function(manifest) {
```

Comment:
We do generally put a space after the `function` keyword (before the `(`) for anonymous functions.